repository-updates.js

```diff
@@ -20,6 +20,33 @@ const getWorkflowRunArtifact = async (context, token, owner, repo, workflowRunId
   await downloadAndUnZip(token, urls[name], name)
 }
 
+const mergeBundle = (gitDir, worktree, bundlePath, refName) => {
+  callGit(['--git-dir', gitDir, 'fetch', bundlePath, refName])
+  // If there is nothing to merge, return early
+  if (callGit(['--git-dir', gitDir, 'rev-list', '--count', `${refName}..FETCH_HEAD`]) === '0') return
+
+  if (worktree) {
+    callGit(['branch', '-M', refName], worktree)
+    callGit(['merge', '--no-edit', 'FETCH_HEAD'], worktree)
+  } else {
+    // If it fast-forwards, we do not need a worktree
+    if (callGit(['--git-dir', gitDir, 'rev-list', '--count', `FETCH_HEAD..${refName}`]) === '0') {
+      callGit(['--git-dir', gitDir, 'update-ref', `refs/heads/${refName}`, 'FETCH_HEAD'])
+    } else {
+      // Fine. We need a worktree. But we don't have one. So let's create one.
+      worktree = `${gitDir}/tmp-worktree`
+      callGit(['--git-dir', gitDir, 'worktree', 'add', '--no-checkout', worktree])
+      // No need for a full checkout, it's not as if we want to resolve any merge conflicts...
+      callGit(['sparse-checkout', 'init'], worktree)
```

Comment:
```suggestion
      callGit(['sparse-checkout', 'set'], worktree)
```

`sparse-checkout init` has been deprecated, but you can use `sparse-checkout set` to get the same result.

Reply:
[Fixed](https://github.com/git-for-windows/git-for-windows-automation/compare/2d306abda1ee728c8294b889610fe762f9d0fbe6..65eb988f268e9b376d05624bdb7a23257d887c9e)!